src/Microsoft.Health.Fhir.Core/Features/Search/SearchValues/DateTimeSearchValue.cs

```diff
@@ -66,15 +66,6 @@ public DateTimeSearchValue(PartialDateTime startDateTime, PartialDateTime endDat
                 defaultFraction: 0.9999999m,
                 defaultUtcOffset: TimeSpan.Zero).ToUniversalTime();
 
```

Comment:
The build is currently failing due to a style cop rule:
`Features\Search\SearchValues\DateTimeSearchValue.cs(69,9): error SA1508: A closing brace should not be preceded by a blank line. [D:\a\1\s\src\Microsoft.Health.Fhir.Core\Microsoft.Health.Fhir.Core.csproj]`